static/app/views/performance/newTraceDetails/traceModels/traceTree.tsx

```diff
@@ -218,6 +218,41 @@ function isJavascriptSDKTransaction(transaction: TraceTree.Transaction): boolean
   );
 }
 
+function isPageloadTransaction(node: TraceTreeNode<TraceTree.NodeValue>): boolean {
```

Comment:
Minor: We should use `TraceTree.Transaction` to type the node params here. Same for line 226.